server/views/referrals/reason.njk

```diff
@@ -19,6 +20,12 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      {% if errors.list.length %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errors.list
+        }) }}
+      {% endif %}
```

Comment:
I wonder if this should be two commits - one to start using the new controller action, and then one to display the errors (both view and controller changes)